src/sessions/session-manager.ts

```diff
@@ -85,14 +97,35 @@ export class SessionManager {
       return;
     }
 
-    delete this.pendingSessions[pendingSession.conversationId];
+    this.pendingSessions = this.pendingSessions.filter(s => s !== pendingSession);
   }
 
-  getSession (params: { conversationId: string }): IExtendedMediaSession {
-    const softphoneHandler = this.getSessionHandler({ sessionType: SessionTypes.softphone }) as SoftphoneSessionHandler;
+  getSession (params: { conversationId: string, sessionType?: SessionTypes, searchScreenRecordingSessions?: boolean }): IExtendedMediaSession {
```

Comment:
I went this route with the `searchScreenRecordingSessions` so if you want to include screen recording sessions in `getSession`, you essentially have to explicitly opt in. This is because as far as I can think of, we only need `getSession` for screen recording for `acceptSession` since you can't update media, mute, hold, or end screen recording sessions.

Doing it this way makes it so we only have to "ask" for the sessionType for `sdk.acceptSession()` and even there it's optional.